src/garvis/github_maintenance.py

```
@@ -0,0 +1,483 @@
+        command: Sequence[str],
+        cwd: Optional[Path] = None,
+    ) -> CommandResult:
+        if not command:
+            raise MaintenancePolicyError("Command must not be empty")
+
+        if any(argument in _BLOCKED_ARGUMENTS for argument in command):
+            raise MaintenancePolicyError(
+                "Force, destructive, or deletion arguments are blocked"
+            )
+
+        completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)